lib/pbench/test/unit/server/test_datasets_daterange.py

```diff
@@ -18,7 +18,9 @@ class TestDatasetsDateRange:
     """
 
     @pytest.fixture()
-    def query_as(self, client, server_config, more_datasets, provide_metadata):
+    def query_as(
+        self, client, server_config, more_datasets, provide_metadata, pbench_admin_token
+    ):
```

Comment:
We're listing the fixtures in the docstring, so you should add `pbench_admin_token` there.  Ditto for the other (three?) `query_as()` functions.